packages/fhir-converter/src/templates/cda/Utils/ReasonForVisit.hbs

```diff
@@ -0,0 +1,12 @@
+{{#with (getFirstCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.12') as |reasons|}}
```

Comment:
utility to extract codes from reason for visit and then is used in Encounters.hbs to hydrate the encounters